### ISSUE TITLE
Regression: Certain models require extra finger bone segments added to be importable

### ISSUE THREAD

**Issue by shiftyscales** [Frooxius/NeosPublic#1713]
## Describe the bug
- Certain models will fail to import properly, and produce an exception in the log
- Reportedly according to gallusfox# 3786 when they added an extra bone segment on the fingers of their avatar it allowed the model to import properly

## Relevant issues
#1691

## To Reproduce
Import the attached reproduction avatar. [Processing_Rigged_Mesh_bug_report_and_model.zip](https://github.com/Frooxius/NeosPublic/files/5929309/Processing_Rigged_Mesh_bug_report_and_model.zip)

## Expected behavior
The model properly imports as on previous versions of Neos

## Bug information (please complete the following information):
 - How often does it happen: Always
 - Does the bug persist after restarting Neos? Yes
 - Neos Version: 2021.2.4.348
 - Neos Platform: Windows
 - Regression: Yes, suspected to have broken in 2021.2.3.616 from resolving issue #1691:
 ```
- Improved hand rig detection, which will now check if finger bone candidates are actually weighted bones on the rig
-- This fixes some models having their fingers bend incorrectly due to detecting proximal bones as metacarpals when there are two finger tips (reported and sample model provided by @Medra)
```

## Additional context
Log file for various model importation attempts by Ian: 
[DESKTOP-IO121IF - 2021.2.4.348 - 2021-02-04 19_16_24.log](https://github.com/Frooxius/NeosPublic/files/5929311/DESKTOP-IO121IF.-.2021.2.4.348.-.2021-02-04.19_16_24.log)

## Reporters:
gallusfox# 3786, with replication case log and sample model provided by Ian Corvid# 8921

**Comment by 3x1t-5tyl3** [comment]
Yeah I got the same issue when importing my model. Not quite sure what happened. I'm happy to give a model to test though.

**Comment by Frooxius** [comment]
I've got a fix for this in the next build already. It's on the danger nightly if you'd like to give it a test or need to import something urgently.

**Comment by Frooxius** [comment]
Fixed in 2021.2.7.1345, thank for report!
